### PR TITLE
assign-roles-to-users

### PR DESCRIPTION
**Pull request by RashmiDholakiya**
Implement assignment of role to users by admin user

### CODE REVIEW

File: app/Http/Controllers/AssignmentController.php [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+namespace App\Http\Controllers;
+
+use App\User;
+use Illuminate\Http\Request;
+
+use App\Http\Requests;
+use App\Http\Controllers\Controller;
+
+class AssignmentController extends Controller
```
Comment (shabbirbharmal) [inline app/Http/Controllers/AssignmentController.php]: Rename it to RoleManagementController

File: app/Http/routes.php [hunk 2]
```diff
@@ -23,6 +23,7 @@
     Route::get('sync-with-jira/{id?}', 'ProjectController@syncWithJira');
     Route::get('ems', 'EmsController@index');
     Route::post('filter', 'FilterController@dateFilter');
+    Route::post('update_role', 'AssignmentController@updateRole');
```
Comment (shabbirbharmal) [inline app/Http/routes.php]: never add underscore to the name of route/url. Change it to update-role

**Comment by shabbirbharmal** [comment]
Please make sure you include ticket number in your branch's name or review title. Check with Karan

File: app/Http/Controllers/AssignmentController.php (hunk 1 above)
Comment (RashmiDholakiya) [inline app/Http/Controllers/AssignmentController.php]: Done

File: app/Http/routes.php (hunk 2 above)
Comment (RashmiDholakiya) [inline app/Http/routes.php]: Done

File: app/Http/Controllers/RoleManagementController.php [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    public function getRoles()
+    {
+        $user = User::paginate(PAGINATE_LIMIT);
+
+        return view('assign.role',compact('user'));
+    }
+
+    public function updateRole(Request $request)
+    {
+        $project = User::findOrFail($request['user_id']);
```
Comment (poojajajal) [inline app/Http/Controllers/RoleManagementController.php]: User::findOrFail won't return project. So why is variable named $project?

**Pull request merged**